src/test/java/io/github/seggan/jyxal/TestHelper.java

```diff
@@ -26,6 +23,7 @@ public static void run(String... args) throws IOException {
             Class<?> clazz = cl.loadClass("jyxal.Main");
             clazz.getMethod("main", String[].class).invoke(null, (Object) new String[0]);
         } catch (ReflectiveOperationException e) {
+            cl.close();
```

Comment:
If you want it this way, put it in a `finally` block and remove both `close()` calls